docs/azure_jumpstart_ag/freezer_monitor/_index.md

```diff
@@ -4,4 +4,98 @@ weight: 100
 toc_hide: true
 ---
 
-# Overview
\ No newline at end of file
+# 
+
+## Overview + Diagram (if applicable)
+
+![Applications and technology stack architecture diagram](./img/placeholder.png)
+
+### Data flow
+
+![Data flow diagram](./img/placeholder.png)
+(not sure if separate dfd is needed with the Diagram above...
+)
+
+## Instructions
+
+### Scenario: Identifying the broken freezer
+
+The manager of the Chicago store has reported that food in one of the freezers has not been staying frozen. She has moved the food to the second freezer and called for service, but the technician reported that the freezer seems to be operating within parameters. Your job as the data analyst is to determine if the temperature sensor in the freezer has observed the issue, and provide the data to the store manager.
+
+- Confirm the issue in Azure Data Explorer
+    1. From your own computer (not the Client VM) Open the dashboard named "Freezer Monitoring" in Azure Data Explorer: https://dataexplorer.azure.com/dashboards
```

Comment:
Assuming these dashboards are already imported by user in ADX, if not we can refer user to import first, at least in the current release.

Reply:
@chintalavr - If they used azd the dashboards will be imported. However, if not (most likely case for now) do you have instructions I can point them to?

Reply:
@chriche-ms @chintalavr we should be crisp here.

Reply:
Followed same pattern used for PoS dashboard so resolving this